ui/src/app/views/registration-page/component.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+
+import {AlertClose, AlertDanger, AlertWarning} from 'app/components/alert';
+import {Button} from 'app/components/buttons';
+import {ResourceCardBase} from 'app/components/card';
+import {baseStyles} from 'app/components/card';
```

Comment:
Nit: I would leave the variable as `styles` in card, and then do this import as the following:
`import {styles as cardStyles} from 'app/components/card';`

Reply:
pushing back on that since this technically is the base style for a card that then for all our other cards in the workbench have styles on top of them (see card.tsx).  in this case i just wanted to adopt a few common things like border radius and shadows, etc